app/routes/interestgroups/components/InterestGroupEdit.js

```diff
@@ -18,11 +18,10 @@ export default class InterestGroupEdit extends Component<{
     return (
       <Content>
         <Helmet title={`Redigerer: ${interestGroup.name}`} />
-        <NavigationTab title={`Redigerer: ${interestGroup.name}`}>
-          <NavigationLink to={`/interestGroups/${interestGroup.id}`}>
-            <i className="fa fa-angle-left" /> Tilbake
-          </NavigationLink>
-        </NavigationTab>
+        <NavigationTab
+          title={`Redigerer: ${interestGroup.name}`}
+          back={{ label: 'Tilbake', path: '/interestGroups/' }}
```

Comment:
This new back goes to `path: '/interestGroups/'` whilst the old one went to `/interestGroups/${interestGroup.id}`. Was this an intentional change?